eo-runtime/src/test/eo/org/eolang/bytes-tests.eo

```diff
@@ -424,8 +424,5 @@
     FF-FF-FF-FF-00-00-00-01
 
 # Test.
-[] > bytes-of-wrong-size-as-number-of-returns-error
-  try > @
-    01-01-01-01.as-number
-    QQ.io.stdout e > [e]
-    false
+[] > throws-on-bytes-of-wrong-size-as-number
+  01-01-01-01.as-number > @
```

Comment:
@maxonfjvipon doesn't look like a correct name of a test, since it doesn't throw anything

Reply:
@yegor256 when test name starts with `throws` a [different](https://github.com/objectionary/eo/blob/c19fd16c8b61e44368af84996ae7ddf30292f534/eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl#L511) block of `to-java` XSL transformation is applied. It checks if an error is thrown. In such case an `error` is returned because 4 bytes [can't be](https://github.com/objectionary/eo/blob/c19fd16c8b61e44368af84996ae7ddf30292f534/eo-runtime/src/main/eo/org/eolang/bytes.eo#L56) converted to number